core/micro-services/workflow-core/src/main/resources/workflow-core-config.yaml

```diff
@@ -0,0 +1,10 @@
+storage:
+  result-storage-mode: mongodb
+  mongodb:
+    url: "mongodb://localhost:27017"
+    database: "texera_storage"
+    commit-batch-size: 1000
+  jdbc:
```

Comment:
why is jdbc configured under storage config? we are not using mysql for storage right?

Reply:
Since the FileResolver is in `workflow-core`, it needs to read the jdbc-related info from the config. Maybe I should put jdbc as a separate section parallel to `storage`.

Also if `storage` section is solely for result storage, how about we rename it to `result-storage`?

Reply:
Keep it as storage. We plan to use it for input and also result